sars2seq/translate.py

```diff
@@ -17,6 +17,75 @@ class StopCodonTooDistantError(TranslationError):
     'The stop codon following the slippery sequence was too far away.'
 
 
+CODONS = {
+    'GCA': 'A',
+    'GCC': 'A',
+    'GCG': 'A',
+    'GCT': 'A',
+    'TGC': 'C',
+    'TGT': 'C',
+    'GAC': 'D',
+    'GAT': 'D',
+    'GAA': 'E',
+    'GAG': 'E',
+    'TTC': 'F',
+    'TTT': 'F',
+    'GGA': 'G',
+    'GGC': 'G',
+    'GGG': 'G',
+    'GGT': 'G',
+    'CAC': 'H',
+    'CAT': 'H',
+    'ATA': 'I',
+    'ATC': 'I',
+    'ATT': 'I',
+    'AAA': 'K',
+    'AAG': 'K',
+    'CTA': 'L',
+    'CTC': 'L',
+    'CTG': 'L',
+    'CTT': 'L',
+    'TTA': 'L',
+    'TTG': 'L',
+    'ATG': 'M',
+    'AAC': 'N',
+    'AAT': 'N',
+    'CCA': 'P',
+    'CCC': 'P',
+    'CCG': 'P',
+    'CCT': 'P',
+    'CAA': 'Q',
+    'CAG': 'Q',
+    'AGA': 'R',
+    'AGG': 'R',
+    'CGA': 'R',
+    'CGC': 'R',
+    'CGG': 'R',
+    'CGT': 'R',
+    'AGC': 'S',
+    'AGT': 'S',
+    'TCA': 'S',
+    'TCC': 'S',
+    'TCG': 'S',
+    'TCT': 'S',
+    'ACA': 'T',
+    'ACC': 'T',
+    'ACG': 'T',
+    'ACT': 'T',
+    'GTA': 'V',
+    'GTC': 'V',
+    'GTG': 'V',
+    'GTT': 'V',
+    'TGG': 'W',
+    'TAC': 'Y',
+    'TAT': 'Y',
+    'TAA': '*',
+    'TAG': '*',
+    'TGA': '*',
+    '---': '-'
+}
```

Comment:
I wouldn't do this. We have a perfectly good (and tested!) dictionary in `dark.aa`, which is safer and also allows you to get rid of 60+ lines:

```python
from dark.aa import CODONS, STOP_CODONS

a = dict([(aa, codon) for aa, codons in CODONS.items() for codon in codons] +
         [(codon, '*') for codon in STOP_CODONS] +
         [('---', '-')])
```

Reply:
Done.